packages/yew/src/dom_bundle/mod.rs

```diff
@@ -76,3 +87,22 @@ impl Bundle {
         self.0.detach(root, parent, parent_to_detach);
     }
 }
+
+#[cfg(feature = "hydration")]
```

Comment:
I'd either move the `use` statements inside the block, or just move the `fn` into the other `impl` block.

Reply:
You need `use super::*` to read Bundle into the namespace.